QUERIES.md

```diff
@@ -40,8 +43,14 @@ REVOKE ALL [PRIVILEGES] ON <db> FROM <user>
 -- revoke all of user's privileges (all DBs and/or cluster admin)
 REVOKE ALL [PRIVILEGES] FROM <user>
 
+-- combine db creation with privilege assignment (user must already exist)
+CREATE DATABASE <name> GRANT <privilege> TO <user>
```

Comment:
I tried this and was only able to get `ALL` to work. Might want to double check whether this is wired up to work with `READ` and `WRITE` privileges.